0000-template.md

```diff
@@ -0,0 +1,73 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Amethyst Issue: (leave this empty)
```

Comment:
If something is being tracked as an informal RFC via a GitHub issue, I would say it should be converted to an RFC. If it is a narrow issue, it can just link to the PR.

Reply:
Yes, all open RFCs in the Amethyst repo should be converted :+1:

Reply:
Can we call it tracking issue and document how that works? I had something like this in mind:

* Pre-RFC, discussion on Discord, etc
* RFC PR
* RFC gets merged
* Tracking issue created with `RFC: approved`

Alternatively, an existing tracking issue can first be labeled `RFC: discussing` and changed later. We should probably make sure that tracking issues only track the implementation progress.

Reply:
Also, should we have a pre-RFC field? I think it would make sense to have a convention that most RFCs are just a topic on Discourse first.

Reply:
Yes, that's why I'd like it to be a convention. Not all RFCs need a pre RFC, but I think very big ones need to collect data berore anything useful can be written down.